client/src/main/java/org/dst/client/cmd/DstCmdStarter.java

```diff
@@ -15,12 +16,30 @@
           commandHandlers = new HashMap<>();
 
   public static void main(String[] args) {
-    //TODO(jyx) Check the server is open or not
 
+    String address;
     if (args.length == 0) {
-      client = new DefaultDstClient(defaultAddress);
```

Comment:
What happens if the default port is occupied

Reply:
I think this problem should be considered by  server.